Menu/GameShell/10_Settings/Wifi/textarea.py

```diff
@@ -19,10 +19,13 @@ class Textarea:
     _BackgroundColor = pygame.Color(229,229,229)
     _CanvasHWND  = None
     _MyWords     = []
+    _BlitWords   = []
     _FontObj     = None
     _LineNumber  = 0
+    _TextLimit   = 63
     _TextFull    = False
```

Comment:
Now that it's trivial to calculate if the text area is full, we could get rid of `self._TextFull` and just check if `len(self._MyWords) >= self._TextLimit`